package.json

```diff
@@ -60,6 +60,7 @@
     "webapi-parser": "^0.5.0"
   },
   "devDependencies": {
+    "@asyncapi/avro-schema-parser": "^3.0.1",
```

Comment:
Added so I can test that the rule against a non default schema Format, there may be another way to do this I am not sure?

Reply:
Dont think so no 😄